test/models/RegExBuilder/Model.test.ts

```diff
@@ -0,0 +1,22 @@
+import { RegExBuilder } from "$models/RegExBuilder";
+
+describe("RegExBuilder", () => {
+  it("validates different phone numbers formats", () => {
+    expect("12345678").toMatch(RegExBuilder.phoneNumber());
+    expect("01123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("1123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("541123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("5491123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("+5491123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("91123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("+19142329901").toMatch(RegExBuilder.phoneNumber());
+    expect("+442087599036").toMatch(RegExBuilder.phoneNumber());
+  });
```

Comment:
Trataria de hacer un `it` case por cada uno, donde en cada uno el mensaje del it explique que testea, porque en general queda raro un it con muchos expects